packages/core/click-handler.js

```diff
@@ -37,25 +37,29 @@ function getResolverUrls(urls) {
       return null;
     }
 
+    // github-search resolver returns a function
     if (typeof url === 'function') {
       return {
         func: url,
       };
     }
 
-    if (typeof url === 'object') {
+    // Live-query resolver results
+    if (url.startsWith('https://githublinker.herokuapp.com')) {
       return {
-        url: url.url.replace('{BASE_URL}', BASE_URL),
```

Comment:
It looks like this change is technically unrelated, but should have been done earlier anyway?

Reply:
Exactly, only the live-query resolver was retuning an object wich didn't have a `{BASE_URL}` placeholder at all.